### DISCUSSION TITLE
Confirmation dialog not showing

### DISCUSSION THREAD

**Discussion by savaged** [Jelmerro/Vieb#502] [Q&A] [answered]
Hi,
I'm using i3wm on Debian 12 and find that I don't see a confirmation box that the Github.com site requires. I do see a toast notification but cannot find how to enter my confirmation.
What am I missing? Or is Vieb missing something?
Thanks in advance
David

**Comment by Jelmerro** [comment, marked as answer]
By default dialogs are blocked as to not interrupt your workflow randomly, as any site can show them as often as they want. This is controlled by the settings named `dialogalert`, `dialogconfirm` and `dialogprompt`. You can also choose to accept the confirm dialog automatically without showing it by doing `dialogconfirm=allow` or `dialogconfirm=allownotify`, but there are many other options for handling them, or you can show them regularly with `dialogconfirm=show`. See `:help dialogconfirm` for more information.

**Reply by savaged** [reply]
Brilliant.
Feels like this might become a FAQ

**Reply by Jelmerro** [reply]
It already is actually :wink:
https://github.com/Jelmerro/Vieb/blob/master/FAQ.md#why-are-prompt-confirm-and-alert-dialogs-not-working

**Reply by savaged** [reply]
Oops, my bad. You've thought of everything ;)
Awesome work!
